extensions/frontend-tiles/src/test/FrontendTiles.test.ts

```diff
@@ -2,222 +2,20 @@
-      async () => {
-        await expectExports(["a"], { iModelId, accessToken });
-        await expectExports(["a", "b"], { iModelId, accessToken, includeIncomplete: true }),
-        await expectExports(["a"], { iModelId, accessToken, includeIncomplete: false });
-      },
-    );
-  });
-});
-
-describe("obtainMeshExportTilesetUrl", () => {
-  before(async () => IModelApp.startup());
-  after(async () => IModelApp.shutdown());
-
-  async function fetchExports(resource: RequestInfo | URL): Promise<Response> {
-    expect(typeof resource).to.equal("string");
-    const url = resource as string;
-
-    let exports: ExportProps[] = [
-      { id: "a", href: "http://tiles.com/a", changesetId: "aaa" },
-      { id: "b", href: "http://tiles.com/b", changesetId: "bbb" },
-      { id: "c", href: "http://tiles.com/c", changesetId: "ccc" },
-    ];
-
-    const result = url.match(/changesetId=([^\&]+)/);
-    if (result) {
-      const changesetId = result[1];
-      exports = exports.filter((x) => x.changesetId === changesetId);
-    }
-
-    return makeExportsResponse({ exports });
-  }
-
-  interface ObtainUrlArgs {
-    id?: string;
-    changesetId?: string;
-    exact?: boolean;
-  }
-
-  async function expectUrl(expected: string | undefined, args: ObtainUrlArgs): Promise<void> {
-    const iModel = new TestConnection(args);
-    await mockFetch(
-      async (resource) => fetchExports(resource),
-      async () => {
-        const url = await obtainMeshExportTilesetUrl({
-          iModel,
-          accessToken,
-          requireExactChangeset: args.exact,
-        });
-
-        expect(url?.toString()).to.equal(expected);
-      },
-    );
-  }
-
-  it("returns undefined if the iModel has no iModelId", async () => {
-    await expectUrl(undefined, { });
-    await expectUrl(undefined, { id: "" });
-  });
-
-  it("selects the first export matching the changeset Id", async () => {
-    await expectUrl("http://tiles.com/a/tileset.json", { id: "imdl", changesetId: "aaa" });
-    await expectUrl("http://tiles.com/b/tileset.json", { id: "imdl", changesetId: "bbb" });
-    await expectUrl("http://tiles.com/c/tileset.json", { id: "imdl", changesetId: "ccc" });
-  });
-
-  it("selects the first export if no export matches the changeset Id", async () => {
-    await expectUrl("http://tiles.com/a/tileset.json", { id: "imdl", changesetId: "bbbbbb" });
-    await expectUrl("http://tiles.com/a/tileset.json", { id: "imdl", changesetId: "bbbbbb", exact: false });
-  });
-
-  it("returns undefined if no export matches the changeset Id and caller requires an exact changeset match", async () => {
-    await expectUrl(undefined, { id: "imdl", changesetId: "bbbbbb", exact: true });
+import { expect } from "chai";
+import { frontendTilesOptions, FrontendTilesOptions, initializeFrontendTiles } from "../FrontendTiles";
+
+describe("FrontendTiles", () => {
```

Comment:
This is not at all an exhaustive test, but this file has now been changed to house test for `FrontendTilesOptions`. Better tests are WIP